interpreter/interpreter.go

```diff
@@ -177,6 +178,31 @@ func (itrp *Interpreter) doStatement(ctx context.Context, stmt semantic.Statemen
 	return nil, nil
 }
 
+// If the option is "now", evaluate the function and store in the execution
+// dependencies.
+func (irtp *Interpreter) evaluateNowOption(ctx context.Context, name string, init values.Value) {
+	if name != "now" {
```

Comment:
We tend to use this string a lot for the same sorts of things. I wonder if it should be a constant somewhere (for our own sake; I assume the compiler interns the string).

Reply:
It is, but trying to import the right package triggers dependency hell. I was torn ... try to fix that in this patch, or leave it as a literal. What do you think?

Reply:
I'm satisfied to know you thought about it. I'd prefer not to unravel the sweater for the sake of this patch.